lib/pbench/test/unit/server/conftest.py

```diff
@@ -52,8 +52,10 @@
 [Indexing]
 index_prefix = unit-test
 
-[keycloak]
-server_url = http://pbench.example.com/
+[authentication]
```

Comment:
You broken some unit tests, because your OIDC code is expecting a `keycloak` section.

Reply:
This brings up an interesting question:  shouldn't we start weaning ourselves off of the on-disk representation of the server configuration for unit tests?

Reply:
FYI: I'm resolving this conversation since Nikhil fixed the problem I pointed out. So if Webb wants to pursue the "config file mocking" issue (and note that in the longer run I'd prefer to remove the config file entirely because specifying some environment variables to a container will be a lot easier to document and manage), then he'll need to create a separate conversation to track it. 😆

Reply:
> Maybe below the line in the context of this PR? 🤔

Quite possibly...but, if not now, when?  😀

In all seriousness, I'm not sure that we want to take on purging the dependency on an on-disk configuration from all the unit tests all at once.  I think it would be better if we started by not making the problem worse...which this PR arguably does.

If we are serious about weaning the Server off of an on-disk config file altogether, I suppose that that will make fixing the unit tests easier, but I'm not convinced that that's the direction we want to go (Nikhil's recent changes being a fine example of the convenience afforded by having the config file...).